crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -255,6 +437,63 @@ def proposals
   api :GET, "/crowbar/:barclamp/1.0/proposals/template",
     "Returns the content of a proposal template"
   param :barclamp, String, desc: "Name of the barclamp", required: true
+  example '
+  {
+    "id": "template-dns",
```

Comment:
that would be too much IMO, as this is just an example. does it really need to be that detailed? It would significantly bloat the LOC of the barclamp_controller